Dockerfile

```diff
@@ -1,15 +1,31 @@
-FROM node:12-alpine
-RUN apk add git && \
-	mkdir -p /app/node_modules && \
-    chown -R node:node /app
-WORKDIR /app
+FROM node:12-stretch AS build
+
+RUN apt-get update && apt-get install -y \
+        g++ \
+        git \
+        make \
+        python \
```

Comment:
I don't believe running these would be necessary, correct? According to https://hub.docker.com/layers/node/library/node/12-stretch/images/sha256-8fa78144d1864f2b08ca4a2d28e9cc32930d78850630652ff338793291a91f0c?context=explore, seems like they already come installed. I can't find python in there but I tried it without python and seems to build fine without explicitly installing it in the Dockerfile

Reply:
Like I said in the beginning, 
> `apt-get update && apt-get install` part can be skipped, the packets are already present in `node:12-stretch`

Absolutely, it is there as a result of my research on the dependencies stack and since the build image is discarded upon final compilation anyways, I've decided to add those just to ensure they are latest versions. No need to explicitly install them.